packages/perseus/src/widgets/interactive-graphs/graphs/components/vector.tsx

```diff
@@ -15,17 +15,11 @@ type Props = {
 
 export function Vector(props: Props) {
     const {tail, tip, color, style} = props;
-
     const [tailPx, tipPx] = useTransformVectorsToPixels(tail, tip);
-    const direction = vec.sub(tip, tail);
+
     return (
         <g style={{stroke: color, strokeWidth: 2}}>
             <SVGLine start={tailPx} end={tipPx} style={style} />
-            <Arrowhead tip={tip} angle={angleDegrees(direction)} />
```

Comment:
Does removing this here not remove the arrows in `movable-line.tsx`?

Reply:
I think it will remove those arrowheads. We should have visual regression tests for this!

I have created a ticket for myself to add those tests.

Reply:
Good catch. It does indeed. Fixing...